thelper/nn/segmentation/deeplabv3.py

```diff
@@ -0,0 +1,56 @@
+import torch
+import torchvision
+import thelper.nn
+
+class deeplabv3_resnet50(thelper.nn.utils.Module):
+
+    # note: this class is just a thin wrapper for deeplabv3_resnet101 (torchvision > 0.6)
+    # https://arxiv.org/abs/1706.05587
```

Comment:
convert to docstring 
```
""" 
This class is a wrapper for `deeplabv3_resnet101`_ (`torchvision >= 0.6` required).

.. _deeplabv3_resnet101: https://arxiv.org/abs/1706.05587
"""
```
So that it gets documented automatically.